src/core/network/createNetwork.js

```diff
@@ -63,7 +63,7 @@ export default (config, logger, lifecycle, networkStrategy) => {
             lifecycle.onBeforeSend({
```

Comment:
@Aaronius @jonsnyder What do you guys think about renaming this hook to `onBeforeNetworkCall` ?

It's really cryptic as it is now since you would need to know that the `Network Gateway` dude is the one triggering this hook, to understand what `send` means.

Reply:
I like it.